components/debug.rst

```diff
@@ -56,6 +57,12 @@ fatal errors)::
 
     ErrorHandler::register();
 
+.. note::
+
+    This one is fine to use in a production environment and will be
```

Comment:
not sure we should tell ppl to register as above in prod, FWBundle explicitly does ` `ErrorHandler::register(null, false)->throwAt(-1, true);``

Reply:
The first argument is `null` by default, as for the second, it tells the error handler to replace the previous one, which people probably want to do if they decide to use ours, right?